README.md

```diff
@@ -1,277 +1,468 @@
-    KnownTokenId,
-    toSDKNumberOrUndefined,
-} from '@brotocol-xyz/bro-sdk';
-import { serializeCVBytes, makeContractCall, broadcastTransaction } from '@stacks/transactions';
-
-// Retrieve bridge information
-const bridgeInfo = await broSdk.bridgeInfoFromStacks({
+> [!WARNING]
+>
+> - `TokenId` values **might change in future updates** (no backward compatibility guaranteed), so you should never cache these values: always get fresh `TokenId`s at runtime using the token address/ticker/runeid, as shown above.
+> - Since known tokens might change at any time, never create `TokenIds` manually - always use SDK methods to ensure validity.
+
+### Supported Routes
+
+```ts
+// Get all possible routes
+const allRoutes = await sdk.getPossibleRoutes();
+
+// Get all possible routes filtered by source chain
+const routesBySourceChain = await sdk.getPossibleRoutes({
+  fromChain: KnownChainId.BRC20.Mainnet,
+});
+
+// Get all possible routes filtered by source and target chain
+const routesBySourceAndTargetChain = await sdk.getPossibleRoutes({
+  fromChain: KnownChainId.BRC20.Mainnet,
+  toChain: KnownChainId.EVM.Ethereum,
+});
+
+// Check if a specific token pair is supported for at least one route
+const isSupported = await sdk.isSupportedRoute({
+  fromChain: KnownChainId.BRC20.Mainnet,
+  toChain: KnownChainId.EVM.Ethereum,
+  fromToken: brc20Token as KnownTokenId.BRC20Token,
+  toToken: evmToken as KnownTokenId.EVMToken,
+});
+
+// If the token pair is supported, get all available routes for that pair
+if (isSupported) {
+  const routesByPair = await sdk.getPossibleRoutes({
+    fromChain: KnownChainId.BRC20.Mainnet,
+    toChain: KnownChainId.EVM.Ethereum,
+    fromToken: brc20Token as KnownTokenId.BRC20Token,
+    toToken: evmToken as KnownTokenId.EVMToken,
+  });
+}
+```
+
+### Basic Operations
+
+The SDK provides three main types of methods for handling cross-chain asset transfers.
+
+- [`bridgeInfoFrom****`](#bridgeinfofrom)
+- [`estimateBridgeTransactionFrom****`](#estimatebridgetransactionfrom)
+- [`bridgeFrom****`](#bridgefrom)
+
+#### `bridgeInfoFrom****`
+
+Retrieves essential data before performing the cross-chain transfer. Purpose:
+
+- Validate whether the route is supported (throws an error if not).
+- Retrieve Brotocol fee values and calculate the exact amount that will arrive on destination chain.
+
+> [!NOTE]
+> These methods do not check the bridge's min/max amount limits. These checks are enforced on-chain, and the transaction will revert if the amount conditions are not met.
+
+Example:
+
+```ts
+import { toSDKNumberOrUndefined } from "@brotocol-xyz/bro-sdk";
+
+// Retrieve bridge info to perform a transfer from Stacks to EVM
+const bridgeInfo = await sdk.bridgeInfoFromStacks({
   fromChain: KnownChainId.Stacks.Mainnet,
   toChain: KnownChainId.EVM.Ethereum,
-  fromToken: KnownTokenId.Stacks.sUSDT,
-  toToken: KnownTokenId.EVM.USDT,
-  amount: toSDKNumberOrUndefined(100),
+  fromToken: stacksToken as KnownTokenId.StacksToken,
```

Comment:
I think it's not a good example, we'd better get stacksToken by `sdk.stacksAddressToStacksToken(...)`

Reply:
I think that's how its was done. I changed the examples to type without `as`.